### PR TITLE
task-lists

### PR DESCRIPTION
**Pull request by MikhailBiltsevich**
I reallized linked lists (single and double)

### CODE REVIEW

File: doublyLinkedList.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
+class DoublyLinkedList {
+  constructor(...values) {
+    for (let i = 0; i < values.length; i += 1) {
+      this.add(values[i]);
```
Comment (AliakseiPuchyk) [inline doublyLinkedList.js:4]: why are you using **_this_** here?

Comment (MikhailBiltsevich) [reply doublyLinkedList.js:4]: > why are you using **_this_** here?

because I call method `add` that located in current class

Comment (AliakseiPuchyk) [reply doublyLinkedList.js:4]: What happens if i create empty DoublyLinkedList and will try add element by index, like this: 

const newDoublyLinkedList = new DoublyLinkedList()
newDoublyLinkedList.addByIndex(1,0)

**Review by AliakseiPuchyk — APPROVED** [review]
Good Work, Mikhail!
Lovely stuff!

File: doublyLinkedList.js (hunk 1 above)
Comment (MikhailBiltsevich) [reply doublyLinkedList.js:4]: I fixed this issue and getNode method, if we use outed of range index for both types of lists

**Pull request merged**